changelogs/fragments/245-ag_listener-ip_address-fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - fixed the expected type of the ip_address parameter to be a list instead of a string (lowlydba.sqlserver.ag_listener)
```

Comment:
```suggestion
  - fixed the expected type of the ip_address, subnet_ip, and subnet_mask parameters to be lists instead of strings (lowlydba.sqlserver.ag_listener)
```